src/App.tsx

```diff
@@ -1,34 +1,84 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
+import './App.css';
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { Todo } from './types/Todo';
+import { User } from './types/User';
+import { getTodos } from './api';
+import { handleFilterTodo } from './services/services';
 
 export const App: React.FC = () => {
+  const [hasModal, setHasModal] = useState(false);
+  const [hasLoader, setHasLoader] = useState(true);
```

Comment:
```suggestion
  const [isLoading, setIsLoading] = useState(false);
```

Better to set `false` as a default value and change it to `true` only before you make API call